pageserver/src/tenant/storage_layer/layer_desc.rs

```diff
@@ -103,6 +103,23 @@ impl PersistentLayerDesc {
         }
     }
 
+    pub fn from_filename(
+        tenant_id: TenantId,
+        timeline_id: TimelineId,
+        filename: LayerFileName,
+        file_size: u64,
+    ) -> Self {
+        match filename {
+            LayerFileName::Image(i) => {
+                // FIXME: can we incremental image layers or is this for future use?
+                Self::new_img(tenant_id, timeline_id, i.key_range, i.lsn, false, file_size)
```

Comment:
I reviewed current `main` and from what I can tell it is always `false`. Let's have a preliminary PR that removes it, and run it by Konstantin and Bojan.

Reply:
Done in #5061.